src/main/java/storagesystem/model/User.java

```diff
@@ -27,24 +25,11 @@ public User(String name, String password, String description, String contactInfo
     }
 
     public User(String name, String description, String contactInformation) {
-        this.name = name;
-        this.description = description;
-        this.contactInformation = contactInformation;
-
-        //set ID and update lastID
-        ID = nextID;
-        nextID++;
+        this(name, "", description, contactInformation);
     }
 
     public User(String name) {
-        this(name, "", "");
-    }
-
-    private User(User userToCopy) {
-        this.name = userToCopy.name;
-        this.description = userToCopy.description;
-        this.contactInformation = userToCopy.contactInformation;
-        this.ID = userToCopy.ID;
+        this(name, "", "", "");
```

Comment:
Should you be able to create a user like this?

Reply:
Mostly just for testing purposes, could be deleted